fetch-csv/src/main.js

```diff
@@ -48,18 +48,18 @@ function getConfig(request) {
   connectorConfig
     .newSelectSingle()
     .setId('textQualifier')
-    .setName('Does the values have a text qualifier?')
+    .setName('Are the values surrounded by single or double quotes?')
     .setAllowOverride(false)
     .addOption(
       connectorConfig
         .newOptionBuilder()
-        .setLabel('No value')
+        .setLabel('No Quotes')
```

Comment:
All 3 options should be consistent in their use of their "quote" or "quotes"

Reply:
Agree. Updated.